### ISSUE TITLE
[FE] Update Error Messages Across Check-in & Pre-Check-in Apps to use standardized h1 tags

### ISSUE THREAD

**Issue by dcgoodwin2112** [department-of-veterans-affairs/va.gov-team#38616]
## User Story
As the Accessibility team, every error or confirmation message to the user should be enclosed in an H1 tag.

## Related Documentation
Refer to this design [story](https://github.com/department-of-veterans-affairs/va.gov-team/issues/38614) for examples (but these wireframes are not all inclusive).

## Description
Update the error and confirmation messages across **all CI & PCI pages** to use the generic h1 styling, rather than the red and green boxes

## Tasks
- [x] Update layout of all error and confirmation messages to be enclosed in an h1 tag (note, updating the component should take care of all of the places where a messaga is displayed but we will need to test all occurrences)
- [ ] Review with Design to ensure that all messages have been updated
- [x] Create or update unit tests
- [x] Create or update e2e tests
- [ ] Update relevant documentations
  - [ ] README
  - [ ] Use Cases

## Acceptance Criteria
- [x] All messages display as an H1
- [x] Passes Accessibility checks using axe
- [x] Deployed to staging.
- [x] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature

**Comment by sarahknoppA6** [comment]
This is currently in prod.
